src/NosCore.Packets/ClientPackets/GuriPacket.cs

```diff
@@ -28,17 +28,17 @@ public class GuriPacket : PacketDefinition
         [PacketIndex(0)]
         public int Type { get; set; }
 
-        [PacketIndex(1)]
+        [PacketIndex(1, IsOptional = true)]
```

Comment:
Argument doesn’t seems to be nullable even in the dance it is set to 1

Reply:
You're right, but there is a problem. There is the special "guri 2" that the server can recieve to start broadcasting the dance. To be sure I have tried this ->
![image](https://gyazo.com/9b5be0b00737ead02b0eee8f84e55cf6)

Reply:
Ok